test/format/format.h

```diff
@@ -154,36 +169,44 @@ extern u_int ntables;
 #define GV(off) (tables[0]->v[V_GLOBAL_##off].v)
 #define GVS(off) \
     (tables[0]->v[V_GLOBAL_##off].vstr == NULL ? "off" : tables[0]->v[V_GLOBAL_##off].vstr)
-#define TV(off) (table->v[V_TABLE_##off].v)
-#define TVS(off) (table->v[V_TABLE_##off].vstr == NULL ? "off" : table->v[V_TABLE_##off].vstr)
+#define NTV(table, off) ((table)->v[V_TABLE_##off].v)
+#define NTVS(table, off) \
+    ((table)->v[V_TABLE_##off].vstr == NULL ? "off" : (table)->v[V_TABLE_##off].vstr)
+#define TV(off) NTV(table, off)
+#define TVS(off) NTVS(table, off)
 
 #define DATASOURCE(table, ds) (strcmp((table)->v[V_TABLE_RUNS_SOURCE].vstr, ds) == 0)
 
 typedef struct {
     WT_CONNECTION *wts_conn;
     WT_CONNECTION *wts_conn_inmemory;
-    WT_SESSION *wts_session;
 
     bool backward_compatible; /* Backward compatibility testing */
     bool configured;          /* Configuration completed */
     bool reopen;              /* Reopen an existing database */
     bool workers_finished;    /* Operations completed */
 
-    char *home;          /* Home directory */
-    char *home_config;   /* Run CONFIG file path */
-    char *home_hsdump;   /* HS dump filename */
-    char *home_key;      /* Key file filename */
-    char *home_pagedump; /* Page dump filename */
-    char *home_stats;    /* Statistics file path */
+    WT_CONNECTION *trace_conn; /* Tracing operations */
+    WT_SESSION *trace_session;
+    WT_SPINLOCK trace_lock;
+    bool trace;
+    bool trace_bulk;
+    bool trace_cursor;
+    bool trace_mirror_fail;
+    bool trace_read;
+    int trace_retain;
```

Comment:
Is this `int` field the reason not to use flags to manage configured tracing options?

Reply:
I believe this is to store the number of trace files to retrain. This is from the readme file `retain=N log files are retained by default, retain N instead.`

Reply:
My question here wasn't clear. I saw the list of bool variables, and wondered why we didn't use flags to track which `trace` options are enabled, as is more WiredTiger-ey. The only thing I spotted was the counter, so I wondered if that was the reason.

My preference would be to turn this into a single flags variable, and a separate counter for trace log file retention:

```
#define FORMAT_TRACE_CURSOR 0x001u
#define FORMAT_TRACE_MIRROR_FAIL 0x002u
...

uint32_t trace_flags;
int trace_retain; /* Count of trace files to retain */
```

Reply:
I agree flags will be more in line with WiredTiger code style. We'll change it.

Reply:
Somehow I messed this comment. Here's a ticket for it [WT-9654](https://jira.mongodb.org/browse/WT-9654)